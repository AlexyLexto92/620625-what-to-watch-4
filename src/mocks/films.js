export const films = [{
  backgroundColor: `#A6B7AC`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/gangs_of_new_york.jpg`,
  description: `In 1862, Amsterdam Vallon returns to the Five Points area of New York City seeking revenge against Bill the Butcher, his father's killer.`,
  director: `Martin Scorsese`,
  genre: `Crime`,
  id: 1,
  isfavorite: false,
  name: `Gangs of new york`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Gangs_of_New_York_Poster.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/gangs_of_new_york.jpg`,
  previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  rating: 8.8,
  released: 2002,
  runTime: 167,
  scoresCount: 370881,
  starring: [`Leonardo DiCaprio`, `Cameron Diaz`, `Daniel Day-Lewis`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#E1DFDE`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/We_need_to_talk_about_Kevin.jpg`,
  description: `Kevin's mother struggles to love her strange child, despite the increasingly dangerous things he says and does as he grows up. But Kevin is just getting started, and his final act will be beyond anything anyone imagined.`,
  director: `Lynne Ramsay`,
  genre: `Drama`,
  id: 2,
  isfavorite: false,
  name: `We need to talk about Kevin`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/We_need_to_talk_about_Kevin.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/we-need-to-talk-about-kevin.jpg`,
  previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  rating: 3.8,
  released: 2011,
  runTime: 112,
  scoresCount: 123240,
  starring: [`Tilda Swinton`, `John C. Reilly`, `Ezra Miller`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#D6CDAF`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Aviator.jpg`,
  description: `A biopic depicting the early years of legendary Director and aviator Howard Hughes' career from the late 1920s to the mid 1940s.`,
  director: `Martin Scorsese`,
  genre: `Drama`,
  id: 3,
  isfavorite: false,
  name: `Aviator`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Aviator.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/aviator.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 9.8,
  released: 2014,
  runTime: 170,
  scoresCount: 307174,
  starring: [`Leonardo DiCaprio`, `Cate Blanchett`, `Kate Beckinsale`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#92918B`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Revenant.jpg`,
  description: `A frontiersman on a fur trading expedition in the 1820s fights for survival after being mauled by a bear and left for dead by members of his own hunting team.`,
  director: `Alejandro G. Iñárritu`,
  genre: `Action`,
  id: 4,
  isfavorite: false,
  name: `The Revenant`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Revenant.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/revenant.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 4,
  released: 2015,
  runTime: 156,
  scoresCount: 618498,
  starring: [`Leonardo DiCaprio`, `Tom Hardy`, `Will Poulter`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#B6A99F`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Fantastic_Beasts.jpg`,
  description: `In an effort to thwart Grindelwald's plans of raising pure-blood wizards to rule over all non-magical beings, Albus Dumbledore enlists his former student Newt Scamander, who agrees to help, though he's unaware of the dangers that lie ahead. Lines are drawn as love and loyalty are tested, even among the truest friends and family, in an increasingly divided wizarding world.`,
  director: `David Yates`,
  genre: `Fantasy`,
  id: 5,
  isfavorite: false,
  name: `Fantastic Beasts: The Crimes of Grindelwald`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Fantastic_Beasts.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 3.4,
  released: 2018,
  runTime: 134,
  scoresCount: 160757,
  starring: [`Eddie Redmayne`, `Katherine Waterston`, `Dan Fogler`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#9B7E61`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/War_of_the_Worlds.jpg`,
  description: `As Earth is invaded by alien tripod fighting machines, one family fights for survival.`,
  director: `Steven Spielberg`,
  genre: `Adventure`,
  id: 6,
  isfavorite: false,
  name: `War of the Worlds`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/War_of_the_Worlds.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/war-of-the-worlds.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 9.3,
  released: 2005,
  runTime: 116,
  scoresCount: 386834,
  starring: [`Tom Cruise`, `Dakota Fanning`, `Tim Robbins`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#73B39A`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/bronson.jpg`,
  description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
  director: `Nicolas Winding Refn`,
  genre: `Action`,
  id: 7,
  isfavorite: false,
  name: `Bronson`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/bronson.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/bronson.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 3.6,
  released: 2008,
  runTime: 92,
  scoresCount: 109661,
  starring: [`Tom Hardy`, `Kelly Adams`, `Luing Andrews`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#977461`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Shutter_Island.jpg`,
  description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
  director: `Martin Scorsese`,
  genre: `Thriller`,
  id: 8,
  isfavorite: false,
  name: `Shutter Island`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Shutter_Island.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/shutter-island.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 4.1,
  released: 2010,
  runTime: 138,
  scoresCount: 1002557,
  starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#CBAC79`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/ones_upon_a_time_in_america.jpg`,
  description: `A former Prohibition-era Jewish gangster returns to the Lower East Side of Manhattan over thirty years later, where he once again must confront the ghosts and regrets of his old life.`,
  director: `Sergio Leone`,
  genre: `Crime`,
  id: 9,
  isfavorite: false,
  name: `Once Upon a Time in America`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Once_Upon_a_Time_in_America.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/Once_Upon_a_Time_in_America.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 9.9,
  released: 1984,
  runTime: 229,
  scoresCount: 276395,
  starring: [`Robert De Niro`, `James Woods`, `Elizabeth McGovern`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
},
{
  backgroundColor: `#A6B7AC`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/gangs_of_new_york.jpg`,
  description: `In 1862, Amsterdam Vallon returns to the Five Points area of New York City seeking revenge against Bill the Butcher, his father's killer.`,
  director: `Martin Scorsese`,
  genre: `Crime`,
  id: 10,
  isfavorite: false,
  name: `Gangs of new york`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Gangs_of_New_York_Poster.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/gangs_of_new_york.jpg`,
  previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  rating: 8.8,
  released: 2002,
  runTime: 167,
  scoresCount: 370881,
  starring: [`Leonardo DiCaprio`, `Cameron Diaz`, `Daniel Day-Lewis`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#E1DFDE`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/We_need_to_talk_about_Kevin.jpg`,
  description: `Kevin's mother struggles to love her strange child, despite the increasingly dangerous things he says and does as he grows up. But Kevin is just getting started, and his final act will be beyond anything anyone imagined.`,
  director: `Lynne Ramsay`,
  genre: `Drama`,
  id: 11,
  isfavorite: false,
  name: `We need to talk about Kevin`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/We_need_to_talk_about_Kevin.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/we-need-to-talk-about-kevin.jpg`,
  previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  rating: 3.8,
  released: 2011,
  runTime: 112,
  scoresCount: 123240,
  starring: [`Tilda Swinton`, `John C. Reilly`, `Ezra Miller`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#D6CDAF`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Aviator.jpg`,
  description: `A biopic depicting the early years of legendary Director and aviator Howard Hughes' career from the late 1920s to the mid 1940s.`,
  director: `Martin Scorsese`,
  genre: `Drama`,
  id: 12,
  isfavorite: false,
  name: `Aviator`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Aviator.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/aviator.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 9.8,
  released: 2014,
  runTime: 170,
  scoresCount: 307174,
  starring: [`Leonardo DiCaprio`, `Cate Blanchett`, `Kate Beckinsale`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#92918B`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Revenant.jpg`,
  description: `A frontiersman on a fur trading expedition in the 1820s fights for survival after being mauled by a bear and left for dead by members of his own hunting team.`,
  director: `Alejandro G. Iñárritu`,
  genre: `Action`,
  id: 13,
  isfavorite: false,
  name: `The Revenant`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Revenant.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/revenant.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 4,
  released: 2015,
  runTime: 156,
  scoresCount: 618498,
  starring: [`Leonardo DiCaprio`, `Tom Hardy`, `Will Poulter`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#B6A99F`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Fantastic_Beasts.jpg`,
  description: `In an effort to thwart Grindelwald's plans of raising pure-blood wizards to rule over all non-magical beings, Albus Dumbledore enlists his former student Newt Scamander, who agrees to help, though he's unaware of the dangers that lie ahead. Lines are drawn as love and loyalty are tested, even among the truest friends and family, in an increasingly divided wizarding world.`,
  director: `David Yates`,
  genre: `Fantasy`,
  id: 14,
  isfavorite: false,
  name: `Fantastic Beasts: The Crimes of Grindelwald`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Fantastic_Beasts.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 3.4,
  released: 2018,
  runTime: 134,
  scoresCount: 160757,
  starring: [`Eddie Redmayne`, `Katherine Waterston`, `Dan Fogler`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#9B7E61`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/War_of_the_Worlds.jpg`,
  description: `As Earth is invaded by alien tripod fighting machines, one family fights for survival.`,
  director: `Steven Spielberg`,
  genre: `Adventure`,
  id: 15,
  isfavorite: false,
  name: `War of the Worlds`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/War_of_the_Worlds.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/war-of-the-worlds.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 9.3,
  released: 2005,
  runTime: 116,
  scoresCount: 386834,
  starring: [`Tom Cruise`, `Dakota Fanning`, `Tim Robbins`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#73B39A`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/bronson.jpg`,
  description: `A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.`,
  director: `Nicolas Winding Refn`,
  genre: `Action`,
  id: 16,
  isfavorite: false,
  name: `Bronson`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/bronson.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/bronson.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 3.6,
  released: 2008,
  runTime: 92,
  scoresCount: 109661,
  starring: [`Tom Hardy`, `Kelly Adams`, `Luing Andrews`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}, {
  backgroundColor: `#977461`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/Shutter_Island.jpg`,
  description: `In 1954, a U.S. Marshal investigates the disappearance of a murderer, who escaped from a hospital for the criminally insane.`,
  director: `Martin Scorsese`,
  genre: `Thriller`,
  id: 17,
  isfavorite: false,
  name: `Shutter Island`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Shutter_Island.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/shutter-island.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 4.1,
  released: 2010,
  runTime: 138,
  scoresCount: 1002557,
  starring: [`Leonardo DiCaprio`, `Emily Mortimer`, `Mark Ruffalo`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
}, {
  backgroundColor: `#CBAC79`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/ones_upon_a_time_in_america.jpg`,
  description: `A former Prohibition-era Jewish gangster returns to the Lower East Side of Manhattan over thirty years later, where he once again must confront the ghosts and regrets of his old life.`,
  director: `Sergio Leone`,
  genre: `Crime`,
  id: 18,
  isfavorite: false,
  name: `Once Upon a Time in America`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Once_Upon_a_Time_in_America.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/Once_Upon_a_Time_in_America.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  rating: 9.9,
  released: 1984,
  runTime: 229,
  scoresCount: 276395,
  starring: [`Robert De Niro`, `James Woods`, `Elizabeth McGovern`],
  videoLink: `http://media.xiph.org/mango/tears_of_steel_1080p.webm`,
}];
export default films;
