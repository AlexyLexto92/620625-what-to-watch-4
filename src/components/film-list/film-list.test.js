import React from 'react';
import renderer from 'react-test-renderer';
import FilmList from './film-list.jsx';
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";
import films from '../../mocks/films.js';
const mockStore = configureStore([]);
const currentFilm = {
  backgroundColor: `#A6B7AC`,
  backgroundImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/background/gangs_of_new_york.jpg`,
  description: `In 1862, Amsterdam Vallon returns to the Five Points area of New York City seeking revenge against Bill the Butcher, his father's killer.`,
  director: `Martin Scorsese`,
  genre: `Crime`,
  id: 1,
  isfavorite: false,
  name: `Gangs of new york`,
  posterImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/poster/Gangs_of_New_York_Poster.jpg`,
  previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/gangs_of_new_york.jpg`,
  previewVideoLink: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  rating: 8.8,
  released: 2002,
  runTime: 167,
  scoresCount: 370881,
  starring: [`Leonardo DiCaprio`, `Cameron Diaz`, `Daniel Day-Lewis`],
  videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`,
};
const isDetails = true;
const onButtonHendler = jest.fn();

it(`FilmList render Filmlist`, () => {
  const store = mockStore({
    filmList: films,
    filteredList: films,
    genres: [`All genres`, `Comedies`, `Crime`, `Documentary`, `Dramas`, `Horror`, `Kids & Family`, `Romance`, `Sci-Fi`, `Thrillers`],
    genreActive: `All genres`,
  });
  const tree = renderer.create(
      <Provider store={store}>
        <FilmList isDetails={isDetails} currentFilm={currentFilm} onButtonHendler={onButtonHendler} />
      </Provider>, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();
  expect(tree).toMatchSnapshot();
});
