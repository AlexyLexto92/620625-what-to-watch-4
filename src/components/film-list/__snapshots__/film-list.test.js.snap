// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmList render Filmlist 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    id={1}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        crossOrigin="anonymous"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[MockFunction]}
      >
        Gangs of new york
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    id={9}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        crossOrigin="anonymous"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[MockFunction]}
      >
        Once Upon a Time in America
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    id={10}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        crossOrigin="anonymous"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[MockFunction]}
      >
        Gangs of new york
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    id={18}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        crossOrigin="anonymous"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[MockFunction]}
      >
        Once Upon a Time in America
      </a>
    </h3>
  </article>
</div>
`;
